config/services/tpay.php

```diff
@@ -33,103 +33,103 @@
+            param('commerce_weavers_sylius_tpay.certificate.cache_ttl_in_seconds'),
         ])
     ;
 
-    $services->set('commerce_weavers_tpay.tpay.security.notification.verifier.checksum', ChecksumVerifier::class)
-        ->alias(ChecksumVerifierInterface::class, 'commerce_weavers_tpay.security.notification.verifier.checksum')
+    $services->set('commerce_weavers_sylius_tpay.tpay.security.notification.verifier.checksum', ChecksumVerifier::class)
+        ->alias(ChecksumVerifierInterface::class, 'commerce_weavers_sylius_tpay.security.notification.verifier.checksum')
     ;
 
-    $services->set('commerce_weavers_tpay.tpay.security.notification.verifier.signature', SignatureVerifier::class)
+    $services->set('commerce_weavers_sylius_tpay.tpay.security.notification.verifier.signature', SignatureVerifier::class)
         ->args([
-            service('commerce_weavers_tpay.tpay.security.notification.resolver.certificate'),
-            service('commerce_weavers_tpay.tpay.security.notification.resolver.trusted_certificate'),
-            service('commerce_weavers_tpay.tpay.security.notification.factory.x509'),
+            service('commerce_weavers_sylius_tpay.tpay.security.notification.resolver.certificate'),
+            service('commerce_weavers_sylius_tpay.tpay.security.notification.resolver.trusted_certificate'),
+            service('commerce_weavers_sylius_tpay.tpay.security.notification.factory.x509'),
         ])
-        ->alias(SignatureVerifierInterface::class, 'commerce_weavers_tpay.security.notification.verifier.signature')
+        ->alias(SignatureVerifierInterface::class, 'commerce_weavers_sylius_tpay.security.notification.verifier.signature')
     ;
 
-    $services->set('commerce_weavers_tpay.tpay.provider.tpay_api_bank_list', TpayApiBankListProvider::class)
+    $services->set('commerce_weavers_sylius_tpay.tpay.provider.tpay_api_bank_list', TpayApiBankListProvider::class)
         ->args([
-            service('commerce_weavers_tpay.tpay.resolver.tpay_transaction_channel_resolver'),
+            service('commerce_weavers_sylius_tpay.tpay.resolver.tpay_transaction_channel_resolver'),
         ])
-        ->alias(TpayApiBankListProviderInterface::class, 'commerce_weavers_tpay.provider.tpay_api_bank_list')
+        ->alias(TpayApiBankListProviderInterface::class, 'commerce_weavers_sylius_tpay.provider.tpay_api_bank_list')
     ;
 
-    $services->set('commerce_weavers_tpay.tpay.resolver.tpay_transaction_channel_resolver', TpayTransactionChannelResolver::class)
+    $services->set('commerce_weavers_sylius_tpay.tpay.resolver.tpay_transaction_channel_resolver', TpayTransactionChannelResolver::class)
         ->args([
             service('payum'),
         ])
-        ->alias(TpayApiBankListProviderInterface::class, 'commerce_weavers_tpay.resolver.tpay_transaction_channel_resolver')
+        ->alias(TpayApiBankListProviderInterface::class, 'commerce_weavers_sylius_tpay.resolver.tpay_transaction_channel_resolver')
```

Comment:
It still seems wrong, e.g. `commerce_weavers_sylius_tpay.tpay.resolver.tpay_transaction_channel_resolver` != `commerce_weavers_sylius_tpay.resolver.tpay_transaction_channel_resolver`.